> BUILD_WITH_CONTAINER

If I unset `BUILD_WITH_CONTAINER` then `make lint` reports

```
Missing license: ./data/charts/istio-telemetry/grafana/fix_datasources.sh
Missing copyright: ./data/charts/istio-telemetry/grafana/fix_datasources.sh
common/Makefile.common.mk:38: recipe for target 'lint-copyright-banner' failed
```

Is there a work item for operator developer instructions?  I don't want to have this conversation here?

--- a/cmd/mesh/manifest-apply.go
+++ b/cmd/mesh/manifest-apply.go
@@ -49,7 +49,7 @@ func addManifestApplyFlags(cmd *cobra.Command, args *manifestApplyArgs) {
 	cmd.PersistentFlags().StringVarP(&args.inFilename, "filename", "f", "", filenameFlagHelpStr)
 	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", "Path to kube config")
 	cmd.PersistentFlags().StringVar(&args.context, "context", "", "The name of the kubeconfig context to use")
-	cmd.PersistentFlags().BoolVar(&args.yes, "yes", false, "Do not ask for confirmation")
+	cmd.PersistentFlags().BoolVarP(&args.yes, "yes", "y", false, "Do not ask for confirmation")
 	cmd.PersistentFlags().DurationVar(&args.readinessTimeout, "readiness-timeout", 300*time.Second, "Maximum seconds to wait for all Istio resources to be ready."+
 		" The --wait flag must be set for this flag to apply")
 	cmd.PersistentFlags().BoolVarP(&args.wait, "wait", "w", false, "Wait, if set will wait until all Pods, Services, and minimum number of Pods "+

--- a/cmd/mesh/manifest-apply.go
+++ b/cmd/mesh/manifest-apply.go
@@ -16,6 +16,7 @@ package mesh
 
 import (
 	"fmt"
+	"io"
 	"os"
 	"strings"
 	"time"
@@ -65,8 +66,10 @@ func manifestApplyCmd(rootArgs *rootArgs, maArgs *manifestApplyArgs) *cobra.Comm
 		Run: func(cmd *cobra.Command, args []string) {
 			l := newLogger(rootArgs.logToStdErr, cmd.OutOrStdout(), cmd.OutOrStderr())
 			if !maArgs.yes && maArgs.kubeConfigPath == "" && maArgs.context == "" {
-				cmd.Print("Supply --kubeconfig, --context, or --yes to apply manifest to cluster\n")
-				os.Exit(1)
+				if !confirm("Are you sure?", cmd.OutOrStdout()) {
+					cmd.Print("Cancelled.\n")
+					os.Exit(1)
+				}
 			}
 			manifestApply(rootArgs, maArgs, l)
 		}}
@@ -115,4 +118,20 @@ func manifestApply(args *rootArgs, maArgs *manifestApplyArgs, l *logger) {
 			l.logAndPrint("Manifest:\n\n", out[cn].Manifest, "\n")
 		}
 	}
+}
+
+func confirm(msg string, writer io.Writer) bool {
+	fmt.Fprintf(writer, "%s ", msg)
+
+	var response string
+	_, err := fmt.Scanln(&response)
+	if err != nil {
+		return false
+	}
+	response = strings.ToUpper(response)
+	if response == "Y" || response == "YES" {
+		return true
+	}
+
+	return false
 }

--- a/cmd/mesh/manifest-apply.go
+++ b/cmd/mesh/manifest-apply.go
@@ -46,13 +46,13 @@ type manifestApplyArgs struct {
 
 func addManifestApplyFlags(cmd *cobra.Command, args *manifestApplyArgs) {
 	cmd.PersistentFlags().StringVarP(&args.inFilename, "filename", "f", "", filenameFlagHelpStr)
-	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", "Path to kube config.")
+	cmd.PersistentFlags().StringVarP(&args.kubeConfigPath, "kubeconfig", "c", "", "Path to kube config")
 	cmd.PersistentFlags().StringVar(&args.context, "context", "", "The name of the kubeconfig context to use")
 	cmd.PersistentFlags().BoolVar(&args.yes, "yes", false, "Do not ask for confirmation")
 	cmd.PersistentFlags().DurationVar(&args.readinessTimeout, "readiness-timeout", 300*time.Second, "Maximum seconds to wait for all Istio resources to be ready."+
-		" The --wait flag must be set for this flag to apply.")
+		" The --wait flag must be set for this flag to apply")
 	cmd.PersistentFlags().BoolVarP(&args.wait, "wait", "w", false, "Wait, if set will wait until all Pods, Services, and minimum number of Pods "+
-		"of a Deployment are in a ready state before the command exits. It will wait for a maximum duration of --readiness-timeout seconds.")
+		"of a Deployment are in a ready state before the command exits. It will wait for a maximum duration of --readiness-timeout seconds")
 	cmd.PersistentFlags().StringSliceVarP(&args.set, "set", "s", nil, setFlagHelpStr)
 }
 

--- a/cmd/mesh/manifest-diff.go
+++ b/cmd/mesh/manifest-diff.go
@@ -51,15 +51,15 @@ func addManifestDiffFlags(cmd *cobra.Command, diffArgs *manifestDiffArgs) {
 			"The format of each list item is \"::\" and the items are comma separated. The \"*\" character represents wildcard selection.\n"+
 			"e.g.\n"+
 			"    Deployment:istio-system:* - compare all deployments in istio-system namespace\n"+
-			"    Service:*:istio-pilot - compare Services called \"istio-pilot\" in all namespaces.")
+			"    Service:*:istio-pilot - compare Services called \"istio-pilot\" in all namespaces")
 	cmd.PersistentFlags().StringVar(&diffArgs.ignoreResources, "ignore", "",
-		"ignoreResources ignores all listed items during comparison. It uses the same list format as selectResources.")
+		"ignoreResources ignores all listed items during comparison. It uses the same list format as selectResources")
 }
 
 func manifestDiffCmd(rootArgs *rootArgs, diffArgs *manifestDiffArgs) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "diff <file|dir> <file|dir>",
-		Short: "Compare manifests and generate diff.",
+		Short: "Compare manifests and generate diff",
 		Long:  "The diff subcommand compares manifests from two files or directories.",
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) != 2 {
@@ -68,30 +68,27 @@ func manifestDiffCmd(rootArgs *rootArgs, diffArgs *manifestDiffArgs) *cobra.Comm
 			return nil
 		},
 		Run: func(cmd *cobra.Command, args []string) {
+			l := newLogger(rootArgs.logToStdErr, cmd.OutOrStdout(), cmd.OutOrStderr())
 			if diffArgs.compareDir {
 				compareManifestsFromDirs(rootArgs, args[0], args[1], diffArgs.selectResources, diffArgs.ignoreResources)
 			} else {
-				compareManifestsFromFiles(rootArgs, args, diffArgs.selectResources, diffArgs.ignoreResources)
+				compareManifestsFromFiles(rootArgs, args, diffArgs.selectResources, diffArgs.ignoreResources, l)
 			}
 		}}
 	return cmd
 }
 
 //compareManifestsFromFiles compares two manifest files
-func compareManifestsFromFiles(rootArgs *rootArgs, args []string, selectResources, ignoreResources string) {
+func compareManifestsFromFiles(rootArgs *rootArgs, args []string, selectResources, ignoreResources string, l *logger) {
 	initLogsOrExit(rootArgs)
 
 	a, err := ioutil.ReadFile(args[0])
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Could not read %q: %v\n", args[0], err.Error())
-		log.Error(err.Error())
-		os.Exit(1)
+		l.logAndFatal(fmt.Sprintf("Could not read %q: %v\n", args[0], err.Error()))
 	}
 	b, err := ioutil.ReadFile(args[1])
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Could not read %q: %v\n", args[1], err.Error())
-		log.Error(err.Error())
-		os.Exit(1)
+		l.logAndFatal(fmt.Sprintf("Could not read %q: %v\n", args[1], err.Error()))
 	}
 
 	diff, err := object.ManifestDiffWithSelectAndIgnore(string(a), string(b), selectResources, ignoreResources)
@@ -102,7 +99,7 @@ func compareManifestsFromFiles(rootArgs *rootArgs, args []string, selectResource
 	if diff == "" {
 		fmt.Println("Manifests are identical")
 	} else {
-		fmt.Printf("Difference of manifests are:\n%s", diff)
+		fmt.Printf("Differences of manifests are:\n%s", diff)
 		os.Exit(1)
 	}
 }
@@ -134,7 +131,7 @@ func compareManifestsFromDirs(rootArgs *rootArgs, dirName1, dirName2, selectReso
 	if diff == "" {
 		fmt.Println("Manifests are identical")
 	} else {
-		fmt.Printf("Difference of manifests are:\n%s", diff)
+		fmt.Printf("Differences of manifests are:\n%s", diff)
 		os.Exit(1)
 	}
 }

--- a/cmd/mesh/manifest-generate.go
+++ b/cmd/mesh/manifest-generate.go
@@ -28,7 +28,7 @@ import (
 type manifestGenerateArgs struct {
 	// inFilename is the path to the input IstioControlPlane CR.
 	inFilename string
-	// outFilename is the path to the generated output filename.
+	// outFilename is the path to the generated output directory.
 	outFilename string
 	// set is a string with element format "path=value" where path is an IstioControlPlane path and the value is a
 	// value to set the node at that path to.
@@ -37,14 +37,14 @@ type manifestGenerateArgs struct {
 
 func addManifestGenerateFlags(cmd *cobra.Command, args *manifestGenerateArgs) {
 	cmd.PersistentFlags().StringVarP(&args.inFilename, "filename", "f", "", filenameFlagHelpStr)
-	cmd.PersistentFlags().StringVarP(&args.outFilename, "output", "o", "", "Manifest output directory path.")
+	cmd.PersistentFlags().StringVarP(&args.outFilename, "output", "o", "", "Manifest output directory path")
 	cmd.PersistentFlags().StringSliceVarP(&args.set, "set", "s", nil, setFlagHelpStr)
 }
 
 func manifestGenerateCmd(rootArgs *rootArgs, mgArgs *manifestGenerateArgs) *cobra.Command {
 	return &cobra.Command{
 		Use:   "generate",
-		Short: "Generates an Istio install manifest.",
+		Short: "Generates an Istio install manifest",
 		Long:  "The generate subcommand generates an Istio install manifest and outputs to the console by default.",
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) != 0 {

--- a/cmd/mesh/manifest-migrate.go
+++ b/cmd/mesh/manifest-migrate.go
@@ -40,13 +40,13 @@ type manifestMigrateArgs struct {
 
 func addManifestMigrateFlags(cmd *cobra.Command, args *manifestMigrateArgs) {
 	cmd.PersistentFlags().StringVarP(&args.namespace, "namespace", "n", defaultNamespace,
-		" Default namespace for output IstioControlPlane CustomResource.")
+		" Default namespace for output IstioControlPlane CustomResource")
 }
 
 func manifestMigrateCmd(rootArgs *rootArgs, mmArgs *manifestMigrateArgs) *cobra.Command {
 	return &cobra.Command{
 		Use:   "migrate [<filepath>]",
-		Short: "Migrates a file containing Helm values to IstioControlPlane format.",
+		Short: "Migrates a file containing Helm values to IstioControlPlane format",
 		Long:  "The migrate subcommand migrates a configuration from Helm values format to IstioControlPlane format.",
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) > 1 {
@@ -100,7 +100,7 @@ func translateFunc(values []byte, l *logger) {
 	}
 	cpYaml, _ := yaml.JSONToYAML([]byte(gotString))
 	if err != nil {
-		l.logAndFatal("error converting json: ", gotString, "\n", err.Error())
+		l.logAndFatal("error converting JSON: ", gotString, "\n", err.Error())
 	}
 	l.print(string(cpYaml) + "\n")
 }

--- a/cmd/mesh/manifest-versions.go
+++ b/cmd/mesh/manifest-versions.go
@@ -38,13 +38,13 @@ type manifestVersionsArgs struct {
 
 func addManifestVersionsFlags(cmd *cobra.Command, mvArgs *manifestVersionsArgs) {
 	cmd.PersistentFlags().StringVarP(&mvArgs.versionsURI, "versionsURI", "u",
-		versionsMapURL, "URI for operator versions to Istio versions map.")
+		versionsMapURL, "URI for operator versions to Istio versions map")
 }
 
 func manifestVersionsCmd(rootArgs *rootArgs, versionsArgs *manifestVersionsArgs) *cobra.Command {
 	return &cobra.Command{
 		Use:   "versions",
-		Short: "List the versions of Istio recommended for and supported by this version of the operator binary.",
+		Short: "List the versions of Istio recommended for and supported by this version of the operator binary",
 		Long:  "List the versions of Istio recommended for and supported by this version of the operator binary.",
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) != 0 {

--- a/cmd/mesh/manifest.go
+++ b/cmd/mesh/manifest.go
@@ -38,7 +38,7 @@ import (
 func ManifestCmd() *cobra.Command {
 	mc := &cobra.Command{
 		Use:   "manifest",
-		Short: "Commands related to Istio manifests.",
+		Short: "Commands related to Istio manifests",
 		Long:  "The manifest subcommand generates, applies, diffs or migrates Istio manifests.",
 	}
 

--- a/cmd/mesh/profile-diff.go
+++ b/cmd/mesh/profile-diff.go
@@ -28,7 +28,7 @@ import (
 func profileDiffCmd(rootArgs *rootArgs) *cobra.Command {
 	return &cobra.Command{
 		Use:   "diff <file1.yaml> <file2.yaml>",
-		Short: "Diffs two Istio configuration profiles.",
+		Short: "Diffs two Istio configuration profiles",
 		Long:  "The diff subcommand displays the differences between two Istio configuration profiles.",
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) != 2 {

--- a/cmd/mesh/profile-dump.go
+++ b/cmd/mesh/profile-dump.go
@@ -44,15 +44,15 @@ type profileDumpArgs struct {
 func addProfileDumpFlags(cmd *cobra.Command, args *profileDumpArgs) {
 	cmd.PersistentFlags().StringVarP(&args.inFilename, "filename", "f", "", filenameFlagHelpStr)
 	cmd.PersistentFlags().StringVarP(&args.configPath, "config-path", "p", "",
-		"The path the root of the configuration subtree to dump e.g. trafficManagement.components.pilot. By default, dump whole tree. ")
+		"The path the root of the configuration subtree to dump e.g. trafficManagement.components.pilot. By default, dump whole tree")
 	cmd.PersistentFlags().BoolVarP(&args.helmValues, "helm-values", "", false,
-		"If set, dumps the Helm values that IstioControlPlaceSpec is translated to before manifests are rendered.")
+		"If set, dumps the Helm values that IstioControlPlaceSpec is translated to before manifests are rendered")
 }
 
 func profileDumpCmd(rootArgs *rootArgs, pdArgs *profileDumpArgs) *cobra.Command {
 	return &cobra.Command{
 		Use:   "dump [<profile>]",
-		Short: "Dumps an Istio configuration profile.",
+		Short: "Dumps an Istio configuration profile",
 		Long:  "The dump subcommand dumps the values in an Istio configuration profile.",
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) > 1 {

--- a/cmd/mesh/profile-list.go
+++ b/cmd/mesh/profile-list.go
@@ -25,7 +25,7 @@ import (
 func profileListCmd(rootArgs *rootArgs) *cobra.Command {
 	return &cobra.Command{
 		Use:   "list",
-		Short: "Lists available Istio configuration profiles.",
+		Short: "Lists available Istio configuration profiles",
 		Long:  "The list subcommand lists the available Istio configuration profiles.",
 		Args:  cobra.ExactArgs(0),
 		Run: func(cmd *cobra.Command, args []string) {

--- a/cmd/mesh/profile.go
+++ b/cmd/mesh/profile.go
@@ -22,7 +22,7 @@ import (
 func ProfileCmd() *cobra.Command {
 	pc := &cobra.Command{
 		Use:   "profile",
-		Short: "Commands related to Istio configuration profiles.",
+		Short: "Commands related to Istio configuration profiles",
 		Long:  "The profile subcommand lists, dumps or diffs Istio configuration profiles.",
 	}
 

--- a/cmd/mesh/root.go
+++ b/cmd/mesh/root.go
@@ -26,8 +26,8 @@ import (
 const (
 	setFlagHelpStr = `Set a value in IstioControlPlane CustomResource. e.g. --set policy.enabled=true.
 Overrides the corresponding path value in the selected profile or passed through IstioControlPlane CR
-customization file.`
-	filenameFlagHelpStr = `Path to file containing IstioControlPlane CustomResource.`
+customization file`
+	filenameFlagHelpStr = `Path to file containing IstioControlPlane CustomResource`
 )
 
 type rootArgs struct {

